blocked-edges/4.18.0-ec.3-CMOSuddenStrictConfigValidation.yaml

```diff
@@ -0,0 +1,16 @@
+to: 4.18.0-ec.3
+from: .*
+# 4.18.0-ec.3 is the only 4.18 (excluding nightlies and CI builds) with strict CMO config validation
+# enabled and that can be reached from versions earlier than 4.17.5, which is meant to help adjust
+# any invalid config in preparation for the 4.18 upgrade.
+fixedIn: 4.18.0-ec.4
+url: https://issues.redhat.com/browse/OCPBUGS-42671
+name: CMOSuddenStrictConfigValidation
+message: |-
+  The Cluster Monitoring Operator (CMO) starts using striter validation for configurations in the 'openshift-monitoring/cluster-monitoring-config' and 'openshift-user-workload-monitoring/user-workload-monitoring-config' ConfigMaps.
+
+  This may result in the operator being marked as `Degraded` and `Unavailable` during the upgrade to `4.18.0-ec.3`, especially if the cluster is upgraded from a `4.17` version that is earlier than `4.17.5`, if any of the ConfigMaps contain invalid configurations.
```

Comment:
```suggestion
  This may result in the operator being marked as `Degraded` and `Unavailable` while updating to a version with strict validation if any of the ConfigMaps contain invalid configurations.
```